sigstore-java/src/main/java/dev/sigstore/trustroot/LegacySigningConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2025 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.trustroot;
+
+import dev.sigstore.trustroot.Service.Config.Selector;
+import java.net.URI;
+import java.time.Instant;
+
+/**
+ * Internal use only: legacy signing config that contains all the necessary information to keep
+ * signers working without signing config being available on a TUF repo
+ */
+public class LegacySigningConfig {
+
+  static final URI REKOR_PUBLIC_GOOD_URI = URI.create("https://rekor.sigstore.dev");
+  static final URI REKOR_STAGING_URI = URI.create("https://rekor.sigstage.dev");
+
+  static final URI FULCIO_PUBLIC_GOOD_URI = URI.create("https://fulcio.sigstore.dev");
+  static final URI FULCIO_STAGING_URI = URI.create("https://fulcio.sigstage.dev");
+
+  static final URI DEX_PUBLIC_GOOD_URI = URI.create("https://oauth2.sigstore.dev/auth");
+  static final URI DEX_STAGING_GOOD_URI = URI.create("https://oauth2.sigstage.dev/auth");
+
+  static final URI TSA_PUBLIC_GOOD_URI = URI.create("https://tsa.sigstore.dev");
```

Comment:
For the TSA, can this be the full path rather than just the host? The TSA client shouldn't assume that the path is the same for all TSAs, so that callers can use other TSAs besides the sigstore/timestamp-authority implementation.